### ISSUE TITLE
Change credential button in proof request does not have accessibility label

### ISSUE THREAD

**Issue by knguyenBC** [bcgov/bc-wallet-mobile#2340]
### Description of problem
With accessibility tools on, proof requests where multiple credentials can satisfy it, the "change credential" button does not have an accessibility label. People are unable to use voice control or voice over to use the function.

### Expected behavior
Add a accessibility label on the "change credential button" to allow people using accessibility tools on their mobile device to use the button

### Steps to reproduce
- Add a prod and test credential (ex. Person from showcase and your own)
- Enable voice control 
- Send a proof request over a connection via mobile verifier asking for full name
- View the proof request and 

### Environment
- [ ] Occurs on Android
- [x] Occurs on iOS

Build #: 1.0.22 build 2138

Android Device Model: 

iOS Device Model: iPhone 12

### Severity
- [ ] High
- [ ] Medium
- [x] Low
